tools/sci/messages_export.py

```diff
@@ -53,6 +54,7 @@ def write_csv(kind, csvdir, entries):
 def messages_export(gamedir, csvdir):
     entries = {}
     for filename in glob.iglob(os.path.join(gamedir, MESSAGES_PATTERN)):
+        print(filename)
```

Comment:
I don't like the printing.
Unix philosophy - keep silent if nothing bad happened.

If you really want it, please add under optional flag.

Reply:
removed